package.json

```diff
@@ -58,15 +58,21 @@
     "@typescript-eslint/eslint-plugin": "^1.5.0",
     "@typescript-eslint/parser": "^1.5.0",
     "babel-loader": "^8.0.5",
+    "connected-react-router": "^6.3.1",
     "eslint-config-prettier": "^4.1.0",
     "eslint-plugin-prettier": "^3.0.1",
+    "history": "^4.7.2",
     "husky": "^1.3.1",
     "lerna": "^3.13.0",
     "lint-staged": "^8.1.4",
     "prettier": "^1.16.4",
     "react": "16.9.0-alpha.0",
     "react-dom": "16.9.0-alpha.0",
+    "react-redux": "^6.0.1",
```

Comment:
we should use version 7.

Reply:
Can you try to update it, and run `yarn start`?

Reply:
Fixed in [3498dd4](https://github.com/iov-one/ponferrada/pull/101/commits/3498dd4c951b81651c765c0a5bf2609f734ee4ca).